pkgs/by-name/sq/sql-studio/package.nix

```
@@ -0,0 +1,45 @@
+{
+  lib,
+  rustPlatform,
+  buildNpmPackage,
+  fetchFromGitHub,
+}:
+let
+  pname = "sql-studio";
+  version = "0.1.27";
+
+  src = fetchFromGitHub {
+    owner = "frectonz";
+    repo = pname;
```

```suggestion
    repo = "sql-studio";
```

See https://github.com/NixOS/nixpkgs/issues/277994